### ISSUE TITLE
NPE using version 11.1.3

### ISSUE THREAD

**Issue by ErwanLeroux** [maximehamm/tzatziki#27]
Hello, the plugin throws an exception using the version 11.1.3, in a project without cucumber :
```
2022-01-03 12:52:39,043 [7162430]  ERROR - tions.RunConfigurationProducer - IntelliJ IDEA 2021.3.1  Build #IU-213.6461.79 
2022-01-03 12:52:39,043 [7162430]  ERROR - tions.RunConfigurationProducer - JDK: 11.0.13; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o. 
2022-01-03 12:52:39,043 [7162430]  ERROR - tions.RunConfigurationProducer - OS: Windows 7 
2022-01-03 12:52:39,046 [7162433]  ERROR - tions.RunConfigurationProducer - Plugin to blame: Cucumber + version: 11.1.3 
2022-01-03 12:52:39,093 [7162480]  ERROR - tions.RunConfigurationProducer - element.parent must not be null 
java.lang.NullPointerException: element.parent must not be null
	at io.nimbly.tzatziki.TzCucumberJavaRunConfigurationProducer.setupConfigurationFromContext(TzCucumberJavaRunConfigurationProducer.kt:46)
	at io.nimbly.tzatziki.TzCucumberJavaRunConfigurationProducer.setupConfigurationFromContext(TzCucumberJavaRunConfigurationProducer.kt:34)
	at com.intellij.execution.actions.RunConfigurationProducer.createConfigurationFromContext(RunConfigurationProducer.java:102)
	at com.intellij.execution.junit.JavaRunConfigurationProducerBase.createConfigurationFromContext(JavaRunConfigurationProducerBase.java:81)
	at com.intellij.execution.actions.RunConfigurationProducer.findOrCreateConfigurationFromContext(RunConfigurationProducer.java:199)
	at com.intellij.execution.actions.PreferredProducerFind.doGetConfigurationsFromContext(PreferredProducerFind.java:110)
	at com.intellij.execution.actions.PreferredProducerFind.getConfigurationsFromContext(PreferredProducerFind.java:93)
	at com.intellij.execution.actions.PreferredProducerFind.findConfigurationFromContext(PreferredProducerFind.java:135)
	at com.intellij.execution.actions.PreferredProducerFind.createConfiguration(PreferredProducerFind.java:32)
	at com.intellij.execution.actions.ConfigurationContext.createConfiguration(ConfigurationContext.java:204)
	at com.intellij.execution.actions.ConfigurationContext.getConfiguration(ConfigurationContext.java:195)
	at com.intellij.sql.annotators.DatabaseLargeFilesAnnotator$RunConfigurationActionFix.createFromContext(DatabaseLargeFilesAnnotator.java:97)
	at com.intellij.sql.annotators.DatabaseLargeFilesAnnotator.annotate(DatabaseLargeFilesAnnotator.java:37)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.runAnnotators(DefaultHighlightVisitor.java:131)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.visit(DefaultHighlightVisitor.java:108)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.runVisitors(GeneralHighlightingPass.java:344)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.lambda$collectHighlights$7(GeneralHighlightingPass.java:283)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.analyzeByVisitors(GeneralHighlightingPass.java:303)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.lambda$analyzeByVisitors$8(GeneralHighlightingPass.java:306)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.analyze(DefaultHighlightVisitor.java:93)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.analyzeByVisitors(GeneralHighlightingPass.java:306)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectHighlights(GeneralHighlightingPass.java:274)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectInformationWithProgress(GeneralHighlightingPass.java:219)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:84)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:56)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:414)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1084)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:407)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:624)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:698)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:646)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:623)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:66)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:406)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:382)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:174)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:181)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:380)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:184)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
2022-01-03 12:52:39,093 [7162480]  ERROR - tions.RunConfigurationProducer - IntelliJ IDEA 2021.3.1  Build #IU-213.6461.79 
2022-01-03 12:52:39,093 [7162480]  ERROR - tions.RunConfigurationProducer - JDK: 11.0.13; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o. 
2022-01-03 12:52:39,093 [7162480]  ERROR - tions.RunConfigurationProducer - OS: Windows 7 
2022-01-03 12:52:39,094 [7162481]  ERROR - tions.RunConfigurationProducer - Plugin to blame: Cucumber + version: 11.1.3 
```

**Comment by maximehamm** [comment]
Thank you Erwan
It will be fixed in 11.1.5
Bonne année !

**Comment by ErwanLeroux** [comment]
Thanks, bonne année :)
